changelog/fragments/validation-bug-bump-api.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      bump api validation library to 431198de9fc2cf82f369efb5c4a90a9cc079a1c3 to
+      fix "CRD key not found" validation bug.
```

Comment:
IMO : It is not very clear to users. I'd recommend something as : 

```suggestion
      fix the error "CRD key not found" faced when the command `bundle validation` is used via upgrading the module api validation used.    
```

Reply:
c/c @estroz @hasbro17

Reply:
This is just a cherry-pick commit. We can add a link to the fixed issue in the v0.18.1 release notes.